cli/support.go

```diff
@@ -109,13 +169,26 @@ func (r *RootCmd) supportBundle() *serpent.Command {
 		},
 	}
 	cmd.Options = serpent.OptionSet{
+		{
+			Flag:          "confirm",
+			FlagShorthand: "y",
+			Env:           "CODER_SUPPORT_BUNDLE_CONFIRM",
+			Description:   "By setting this to true, you confirm that you will treat the resulting support bundle as if it contained sensitive information.",
```

Comment:
```suggestion
			Description:   "By setting this, you confirm that you will treat the resulting support bundle as if it contained sensitive information.",
```
As a flag, you don't necessarily "set to true", you just provide it. Up to you if you want to change this.